gateway/intents.go

```diff
@@ -12,7 +12,7 @@ type Intents uint32
 const (
 	IntentGuilds Intents = 1 << iota
 	IntentGuildMembers
-	IntentGuildBans
```

Comment:
Can we add an alias to avoid breaking old code?

```go
// IntentGuildBans aliases to IntentGuildModeration.
//
// Deprecated: IntentGuildModeration is the more correct constant to use.
const IntentGuildBans = IntentGuildModeration